lib/Bindings/Python/OMModule.cpp

```diff
@@ -366,6 +366,15 @@ Map::dunderGetItem(std::variant<intptr_t, std::string, MlirAttribute> key) {
 // Convert a generic MLIR Attribute to a PythonValue. This is basically a C++
 // fast path of the parts of attribute_to_var that we use in the OM dialect.
 static PythonPrimitive omPrimitiveToPythonValue(MlirAttribute attr) {
+  if (mlirAttributeIsAInteger(attr)) {
+    MlirType type = mlirAttributeGetType(attr);
+    if (mlirTypeIsAIndex(type) || mlirIntegerTypeIsSignless(type))
+      return py::int_(mlirIntegerAttrGetValueInt(attr));
+    if (mlirIntegerTypeIsSigned(type))
+      return py::int_(mlirIntegerAttrGetValueSInt(attr));
+    return py::int_(mlirIntegerAttrGetValueUInt(attr));
+  }
```

Comment:
This is copied from: https://github.com/llvm/llvm-project/blob/1a9acd786d493b00c08d1611f51420d421b74cf1/mlir/lib/Bindings/Python/IRAttributes.cpp